README.md

```diff
@@ -0,0 +1,43 @@
+## Edit Discourse posts locally
+Discedit allows you to edit [Discourse](https://www.discourse.org/) posts in your favourite local text editor. It woks by pulling a post from Discourse, openning it in your favourite text editor and automatically pushing the edits. 
+
+## Setting up
+You will need to install [Go](https://golang.org/doc/install) and generate a personal global Discourse API key. If you don't have the rights to handle keys, talk to the administrator of your Discourse installation.
+
+1. Clone this repository and navigate into the directory:
+
+`git clone https://github.com/niemeyer/discedit.git`
+
+`cd discedit`
+
+2. Compile the application:
+
+`go build`
+
+3. Create a `.discedit` file (note the dot before `discedit`) in your `$HOME` directory and add the following:
```

Comment:
Let's please replace the rest of this file from here on with something that only says something along the lines of:
```
3. Run `./discedit <topic URL>` and follow the tool instructions to complete your setup.
```
As this means we'll never have this README out of date.

As an aside, conventionally we use `<this>` for things that need to be there, and `[this]` for things that might be there or not. And if it's a positional argument, we tend to do the latter with both, like `[<this>]`.